src/main/src/jetty/geoserver_data/demo/WCS_describeCoverage.xml

```diff
@@ -0,0 +1,19 @@
+<!-- A sample describe request.  The schema is generated automatically by  -->
```

Comment:
I was undecided - it gives you the example requests in GeoServer which I used a bit when trying out WPS requests.  I thought these may be useful - but no - technically not needed